### PR TITLE
added varian modal for co-op

### LINKED ISSUE #83: Varian Modal
Add new modal for [Varian Medical Systems](https://www.varian.com/) under the http://cs.umanitoba.ca/~wics/co-op/ page

Company link: https://www.varian.com/

### PR DESCRIPTION
**Pull request by darylfung96**
resolve #83

### CODE REVIEW

File: _co-op-companies/N-varian [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: "Varian"
+identification: "varian"
+description: "Varian uses intelligent approach to fight against cancer."
+link: "https://www.varian.com/"
+image: "assets/img/logos/varian.png"
+width: "200px"
+complete: false
```
Comment (shenyunw) [inline _co-op-companies/N-varian]: change complete to true for modal to show

Comment (darylfung96) [reply _co-op-companies/N-varian]: okay it is done

File: _co-op-companies/N-varian.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: "Varian"
+identification: "varian"
+description: "Varian uses intelligent approach to fight against cancer."
```
[the remark this answers is not in the document]
Comment (darylfung96) [reply _co-op-companies/N-varian.md]: It's updated

**Comment by jadasc** [comment]
Do you mind changing the logo to this one? Please and thank youuuuu

Varian

File: _co-op-companies/N-varian.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+title: "Varian Medical Systems
```
Comment (jadasc) [inline _co-op-companies/N-varian.md]: I think we're missing the closing double quote `"`

**Comment by darylfung96** [comment]
> Do you mind changing the logo to this one? Please and thank youuuuu
> 
> Varian

Done

File: _co-op-companies/N-varian.md (hunk 3 above)
Comment (darylfung96) [reply _co-op-companies/N-varian.md]: Done

File: _co-op-companies/N-varian.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+---
+title: "Varian"
```
[the remark this answers is not in the document]
Comment (darylfung96) [reply _co-op-companies/N-varian.md]: It is updated

**Pull request merged**